src/applications/personalization/profile/components/direct-deposit/DirectDepositContent.jsx

```diff
@@ -288,83 +295,122 @@ export const DirectDepositContent = ({
     },
   ];
 
-  // Render nothing if the user should not see the Direct Deposit feature.
+  const mfaHandler = isAuthenticatedWithSSO => {
```

Comment:
This logic is taken directly from `src/applications/personalization/profile/components/account-security/TwoFactorAuthorizationStatus.jsx` where we also should a button to set up 2 factor authentication.